src/nrnoc/nrn_filesystem.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+/**
+ * @brief Header wrapping <filesystem> to help us use it with older compilers.
+ *
+ * It should be safe-ish to use this inside functions, but using it in interfaces is almost
+ * certainly a bad idea.
+ */
+#if __has_include(<filesystem>)
+#include <filesystem>
+namespace neuron::std {
+namespace filesystem = ::std::filesystem;
+}
+#else
+#include <experimental/filesystem>
```

Comment:
Just a thought, I remember discussing about this some time ago. Maybe we should drop `experimental` handling and clearly state we require a "complete" C++17 compiler. Manylinux wise I think that's ok if we bump to GCC10 (#2140)

Reply:
Fine by me, but I'd rather that happens separately. Juggling too many things already!